cmd/utils/flags.go

```diff
@@ -361,6 +361,31 @@ var (
 		Usage: "Suggested gas price base correction factor (%)",
 		Value: 110,
 	}
+	GlsFlag = cli.StringFlag{
+		Name:  "gls",
+		Usage: "Gas limit adjustment strategy ( none | target | blkutil )",
+		Value: "target",
+	}
+	GlsTargetFlag = cli.StringFlag{
+		Name:  "glstarget",
+		Usage: "Constant gas limit target",
+		Value: "3141592",
+	}
+	GlsBlkUtilFlag = cli.IntFlag{
+		Name:  "glsblkutil",
+		Usage: "Block utilization target (%)",
+		Value: 67,
+	}
+	GlsMinFlag = cli.StringFlag{
+		Name:  "glsmin",
+		Usage: "Gas limit adjustment range minimum (applies to all strategies)",
+		Value: "1000000",
+	}
+	GlsMaxFlag = cli.StringFlag{
+		Name:  "glsmax",
+		Usage: "Gas limit adjustment range maximum (applies to all strategies)",
+		Value: "10000000",
```

Comment:
I'm a bit scared about the block utilization strategy. Gas prices can stay quite low when filling 70-80% of the blocks so if people start to use the network more heavily, they could kill it if miners are using blkutil strategy with no upper limit.
If the network runs at 10 million gas limit without any problems and the only limiting factor is lazy miners not raising this parameter, we can raise it in our next update.